micromagneticmodel/driver.py

```diff
@@ -325,3 +330,24 @@ def _setup_working_directory(system, dirname, mode, append=True):
         workingdir = system_dir / f"{mode}-{next_number}"
         workingdir.mkdir(parents=True)
         return workingdir
+
+    @staticmethod
+    def _conversion_to_hms(time_dfference):
+        total_seconds = int(time_dfference.total_seconds())
```

Comment:
```suggestion
        total_seconds = round(time_dfference.total_seconds())
```

Reply:
Alternatively, we could use `ceil` so that any simulation takes at least 1 second from Ubermag's perspective. Then we could remove the `if` case below and shorter simulations are anyway not realistic.

What do you think?